WooCommerce/Classes/ViewRelated/Products/ProductSelector/ProductVariationSelectorViewModel.swift

```diff
@@ -16,6 +16,11 @@ final class ProductVariationSelectorViewModel: ObservableObject {
     ///
     private let stores: StoresManager
 
+    /// Determines if product variations should be toggled on selection
+    /// `false` by default, mainly used to allow multi-seleciton on the Order Creation flow
+    ///
+    let shouldToggleItemOnSelection: Bool
```

Comment:
It seems like we are duplicating logic here, can we somehow have or extract the logic to a common class for both?

Reply:
> It seems like we are duplicating logic here, can we somehow have or extract the logic to a common class for both?

I'd prefer to not change this part within the scope of this PR, as the behaviour might be different for the product selector and the product variation selector, specially since is also used with Coupons. 

Happy to revisit it outside of this PR though, for example I think we could pass some sort of common Configuration class to the initializer, or use generics to abstract some bits of the logic on both view models. Opened a new issue to keep track of this : https://github.com/woocommerce/woocommerce-ios/issues/8988